packages/fetch/src/index.ts

```diff
@@ -0,0 +1,162 @@
+  generateVerbImports,
+  GeneratorDependency,
+  GeneratorOptions,
+  GeneratorVerbOptions,
+  GetterPropType,
+  stringify,
+  toObjectString,
+  generateBodyOptions,
+  isObject,
+} from '@orval/core';
+
+const PARAMS_SERIALIZER_DEPENDENCIES: GeneratorDependency[] = [
+  {
+    exports: [
+      {
+        name: 'qs',
+        default: true,
+        values: true,
+        syntheticDefaultImport: true,
+      },
+    ],
+    dependency: 'qs',
+  },
+];
+
+export const getDependencies: ClientDependenciesBuilder = (
+  hasParamsSerializerOptions: boolean,
+) => [...(hasParamsSerializerOptions ? PARAMS_SERIALIZER_DEPENDENCIES : [])];
+
+const generateRequestFunction = (
+  {
+    queryParams,
+    operationName,
+    response,
+    body,
+    props,
+    verb,
+    formData,
+    formUrlEncoded,
+    override,
+  }: GeneratorVerbOptions,
+  { route }: GeneratorOptions,
+) => {
+  const isRequestOptions = override?.requestOptions !== false;
+  const isFormData = override?.formData !== false;
+  const isFormUrlEncoded = override?.formUrlEncoded !== false;
+
+  const getUrlFnName = camel(`get-${operationName}-url`);
+  const getUrlFnProps = toObjectString(
+    props.filter(
+      (prop) =>
+        prop.type === GetterPropType.PARAM ||
+        prop.type === GetterPropType.NAMED_PATH_PARAMS ||
+        prop.type === GetterPropType.QUERY_PARAM,
+    ),
+    'implementation',
+  );
+  const getUrlFnImplementation = `export const ${getUrlFnName} = (${getUrlFnProps}) => {
+${
+  queryParams
+    ? `
+  const normalizedParams = new URLSearchParams();
+
+  Object.entries(params || {}).forEach(([key, value]) => {
+    if (value === null) {
+      normalizedParams.append(key, 'null');
+    } else if (value !== undefined) {
+      normalizedParams.append(key, value.toString());
+    }
+  });`
+    : ''
+}
+
+  return \`${route}${queryParams ? '?${new URLSearchParams(normalizedParams).toString()}' : ''}\`
+}\n`;
+  const getUrlFnProperties = props
+    .filter(
+      (prop) =>
+        prop.type === GetterPropType.PARAM ||
+        prop.type === GetterPropType.QUERY_PARAM ||
+        prop.type === GetterPropType.NAMED_PATH_PARAMS,
+    )
+    .map((param) => {
+      if (param.type === GetterPropType.NAMED_PATH_PARAMS) {
+        return param.destructured;
+      } else {
+        return param.name;
+      }
+    })
+    .join(',');
+
+  const args = `${toObjectString(props, 'implementation')} ${isRequestOptions ? `options?: RequestInit` : ''}`;
+  const retrunType = `Promise<${response.definition.success || 'unknown'}>`;
+
+  const globalFetchOptions = isObject(override?.requestOptions)
+    ? `${stringify(override?.requestOptions)?.slice(1, -1)?.trim()}`
+    : '';
+  const fetchMethodOption = `method: '${verb.toUpperCase()}'`;
+
+  const requestBodyParams = generateBodyOptions(
+    body,
+    isFormData,
+    isFormUrlEncoded,
+  );
+  const fetchBodyOption = requestBodyParams
+    ? `body: JSON.stringify(${requestBodyParams})`
```

Comment:
In this case we only handle object body no?

Reply:
Yes, Here we process and request only the body parameters.

Reply:
What I wanted to say is that for now if we have a binary in body it won't work. Is it what you want?

Reply:
@anymaniax 

Support for binary was not included in the scope.
For the functionality I want at this point, just being able to send a simple json request parameter is sufficient.
After that, I think it would be good if other users, including other users, could receive support when they have a case or improvement idea.